components/Layout/index.js

```diff
@@ -0,0 +1,12 @@
+import Navigation from "../Navigation";
+import Header from "../Header";
+
+export default function Layout({ children }) {
+  return (
+    <>
+      <Header />
+      <main>{children}</main>
+      <Navigation />
```

Comment:
when you scroll down to the bottom of the list, your last item seems to overlap with the navigation and the categories are displayed behind the navigation